### PR TITLE
POL-86 FSM Suspicious Users

### PR DESCRIPTION
**Pull request by dfrankel33**
### Description

initial release of Suspicious Users policy

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: saas/fsm/suspicious_users/fsm-suspicious_users.pt [hunk 1]
```diff
@@ -0,0 +1,149 @@
+name "SaaS Manager - Suspicious Users"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy will raise an alert when Flexera SaaS Manager identifies suspicious users logging into SaaS applications. See the [README](https://github.com/flexera/policy_templates/tree/master/saas/fsm/suspicious_users/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```
Comment (cdwilhelm) [inline saas/fsm/suspicious_users/fsm-suspicious_users.pt]: product doesn't want to use alert.  instead change **raise an alert** to **create an incident**.

File: saas/fsm/suspicious_users/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# SaaS Manager - Suspicious Users
+
+## What it does
+
+This policy will raise an alert when Flexera SaaS Manager identifies suspicious users logging into SaaS applications.
```
Comment (cdwilhelm) [inline saas/fsm/suspicious_users/README.md]: product doesn't want to use alert.  instead change **raise an alert** to **create an incident**.

File: saas/fsm/suspicious_users/fsm-suspicious_users.pt [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+           field "unauthorized", val(col_item, "unauthorized")
+        end
+    end
+end
+
+datasource "ds_users_cleanup" do
+  run_script $js_users_cleanup, $ds_suspicious_users
+end
+
+script "js_users_cleanup", type: "javascript" do
```
Comment (cdwilhelm) [inline saas/fsm/suspicious_users/fsm-suspicious_users.pt]: apply some sorting, maybe by Email address.

Comment (dfrankel33) [reply saas/fsm/suspicious_users/fsm-suspicious_users.pt]: this isn't the point of the policy.  there should never be any users returned from this call. if there are, it should raise an incident

Comment (cdwilhelm) [reply saas/fsm/suspicious_users/fsm-suspicious_users.pt]: If there is a list of users shouldn't the list be sorted in some way?

Comment (dfrankel33) [reply saas/fsm/suspicious_users/fsm-suspicious_users.pt]: i don't see why.  there is no way to identify priority, and every user in this list should be included in the incident

**Pull request merged**